app-backend/db/src/main/scala/com/azavea/rf/database/filters/Filterables.scala

```diff
@@ -108,7 +110,7 @@ trait Filterables extends RFMeta {
           case (Some(bboxPolygons), _) => {
             val fragments = bboxPolygons.map(bbox =>
               fr"ST_Intersects(data_footprint, ${bbox})")
-            Some(Fragments.and(fragments: _*))
+            Some(fr"(" ++ Fragments.or(fragments: _*) ++ fr")")
```

Comment:
This seems fine not to test directly, since it's exercised in your testing instructions and will be really obvious if it breaks